scripts/migration/migrate_registered_meta.py

```diff
@@ -0,0 +1,77 @@
+""" Changes existing registered_meta on a node to new schema layout
+required for the prereg-prize
+"""
+import re
+import ast
+import sys
+import logging
+
+from nose.tools import *
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def get_registered_nodes():
+    return models.Node.find(
+        Q('is_registration', 'eq', True)
+    )
+
+
+def main(dry_run=True):
+    init_app(routes=False)
+    count = 0
+
+    if not dry_run:
+        scripts_utils.add_file_logger(logger, __file__)
+        logger.info("Iterating over all registrations")
+        models.MetaSchema.remove()
+
+    for node in get_registered_nodes():
+        for schema in node.registered_meta:
+            values = ast.literal_eval(node.registered_meta.get(schema))
+
+            valid_schema = {
+                'embargoEndDate': values['embargoEndDate'],
+                'registrationChoice': values['registrationChoice']
+            }
+
+            # in most schemas, answers are just stored as { 'itemX': 'answer' }
+            matches = dict()
+            for val in values:
+                m = re.search("item[0-9]*", val)
+                if m is not None or val == 'datacompletion' or val == 'summary':
```

Comment:
Why are `'datacompletion' and `'summary'` special-cased here? Would be good to have a comment explaining this.

Might be good to save these special cases in a `set` and check membership in it. 

```python
if match is not None and match not in NON_ITEMS:
```

Of course, you'll come up with a more descriptive name that `NON_ITEMS` though. =)